client/js/lounge.js

```diff
@@ -417,7 +419,7 @@ $(function() {
 		if (chan.hasClass("lobby")) {
 			cmd = "/quit";
 			var server = chan.find(".name").html();
-			if (!confirm("Disconnect from " + server + "?")) {
+			if (!confirm(t.translate("disconnect_from", {server: server}))) {
```

Comment:
should this be `client.disconnect_from` ?

Reply:
Yep